yarn.lock

```diff
@@ -6774,7 +6774,7 @@ sprintf-js@~1.0.2:
   resolved "https://codeload.github.com/scality/sproxydclient/tar.gz/e45c547833f03b1fa0409e0499433dcb386442f5"
   dependencies:
     async "^3.1.0"
-    werelogs scality/werelogs#351a2a3
+    werelogs scality/werelogs#8.1.0
```

Comment:
Same concern here - we somehow went from that same commit to 8.1.0? But 8.1.0 is 2 years ahead time-wise

```
git show tags/8.1.0

tag 8.1.0
Tagger: Rahul Padigela <rahul.padigela@scality.com>
Date:   Fri Apr 9 17:02:24 2021 -0700

v8.1.0

commit e8f8287 (tag: 8.1.0)
Author: Rahul Padigela <rahul.padigela@scality.com>
Date:   Fri Apr 9 17:02:23 2021 -0700

    v8.1.0
```